netmanager/src/views/pages/DeployDevice/index.js

```
@@ -0,0 +1,513 @@
+  CardContent,
+  CardHeader,
+  Divider,
+  Checkbox,
+  FormControlLabel,
+  Snackbar,
+  IconButton
+} from '@material-ui/core';
+import { Alert } from '@material-ui/lab';
+import CloseIcon from '@material-ui/icons/Close';
+import OutlinedSelect from '../../components/CustomSelects/OutlinedSelect';
+import { Map, TileLayer, Marker, Popup } from 'react-leaflet';
```

Biome

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)